src/internal/ans.ts

```diff
@@ -365,7 +386,7 @@ export async function getName(args: {
     res = sanitizeANSName(res);
   }
 
-  return res;
+  return isActiveANSName(res) ? res : undefined;
```

Comment:
dow e need to update this function test accordingly?

Reply:
We deploy ANS to the localnet as part of the E2E test suite, maybe now is the time to switch the test to use that.